content/applications/productivity/knowledge/management.rst

```diff
@@ -0,0 +1,118 @@
+:guilabel:`+` button next to a category or another article.
+
+.. tip::
+   Shortcut the creation of private articles; press **alt** and **c** simultaneously on your
+   keyboard.
+
+.. _management/share:
+
+Sharing
+-------
+
+You can share articles with internal or external users. To do so, open the share menu by clicking
+:guilabel:`Share` in the top-right menu of articles.
+
+Invite users
+~~~~~~~~~~~~
+
+To share articles with specific users (internal or external, such as a partner or a customer), click
+:guilabel:`Invite`. This opens a pop-up window in which you can choose the :guilabel:`Permission`
+:ref:`(i.e, access rights) <management/categories>` and enter the :guilabel:`Recipients`' name or
+email.
+
+You can also restrict a specific user from accessing the article by selecting :guilabel:`No access`
+permission.
+
+.. image:: management/invite.png
+   :alt: pop-up window to invite users to access a Knowledge article
+
+Share online
+~~~~~~~~~~~~
+
+To share articles **online**, activate the :guilabel:`Share to web` button. Doing so generates a URL
+link anyone can use to view the article.
+
+Additionally, the share menu displays the default permission for internal members along with all the
+users who have been granted specific permission.
+
+.. image:: management/share-menu.png
+   :alt: Sharing menu with URL for online sharing and the list of members.
+
+.. _management/remove:
+
+Removal
+-------
+
+To remove an article, you can either **delete** it or **archive** it.
+
+To delete an article, open it and click the vertical ellipsis button :menuselection:`(⋮) -->
+Delete`. The article is moved to the trash for 30 days before being permanently deleted. To restore
+it, click :guilabel:`Open the Trash`, select the article, and click :guilabel:`Restore`.
+
+To archive articles, click :guilabel:`Search`, select the article(s), and click
+:menuselection:`Action --> Archive --> Archive`. Archived articles are hidden from the search menu.
+To retrieve an archived article, add a custom filter to display them (:menuselection:`Search -->
+Filters --> Add Custom Filter`, and set :guilabel:`Active` as :guilabel:`is No`). Then, select the
+article(s) and go to :menuselection:`Action --> Unarchive`.
+
+.. _management/structure:
+
+Structure of articles
+=====================
+
+Articles are organized into a hierarchical structure wherein the article on top is a parent article,
+and those underneath are called nested articles. This structure allows the grouping of related
+articles.
+
+To establish this hierarchy, create new articles by clicking the :guilabel:`+` button next to the
+parent-to-be article, or move existing articles by either dragging and dropping them under the
+parent-to-be or by clicking the vertical ellipsis button (:guilabel:`⋮`), clicking :guilabel:`Move
+To` on the toolbox and selecting the article to use as a parent.
+
+.. note::
+   The number of nested articles is **not** limited.
```

Comment:
Would people expect a limit? It would be more important to mention if there was indeed a limit. Consider removing.